verticaltabs.js

```diff
@@ -288,7 +304,7 @@ VerticalTabs.prototype = {
                       window.VerticalTabs.numPinnedtabs();
         break;
       case this.closingTabsEnum.OTHER:
-        tabsToClose = this.visibleTabs.length - 1 - window.VerticalTabs.numPinnedtabs();
+        tabsToClose = this.window.gBrowser.visibleTabs.length - 1 - window.VerticalTabs.numPinnedtabs();
```

Comment:
Is `this.window` different than `window`?  And if it isn't, can we just use `window` to match the rest of this line?